Database/Entities/AdminNoteEntityBase.cs

```diff
@@ -0,0 +1,21 @@
+using System.ComponentModel.DataAnnotations.Schema;
+using Database.Entities.Interfaces;
+
+namespace Database.Entities;
+
+/// <summary>
+/// Base class for an entity that serves as an admin note for an <see cref="IAdminNotableEntity"/>
+/// </summary>
+public abstract class AdminNoteEntityBase : UpdateableEntityBase, IAdminNoteEntity
+{
+    [Column("note")]
+    public string Note { get; set; } = string.Empty;
```

Comment:
Do we want a size constraint on this?

Reply:
Probably not. Mania modding posts incoming lol